ironfish/src/mining/manager.ts

```diff
@@ -84,15 +79,15 @@ export class MiningManager {
       // Send an initial block template to the requester so they can begin working immediately
       void this.chain.getBlock(this.chain.head).then((currentBlock) => {
         if (currentBlock) {
-          this.onConnectedBlock(currentBlock)
+          void this.onConnectedBlock(currentBlock)
```

Comment:
I think we might need a catch statement here and on `this.chain.onConnectBlock.on`. If there's an error in this function not positive but I think it could crash the node as an unhandled promise exception